freqtrade/strategy/interface.py

```diff
@@ -692,7 +697,12 @@ def analyze_ticker(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
         dataframe = self.advise_exit(dataframe, metadata)
         return dataframe
 
-    def _analyze_ticker_internal(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+    def _analyze_ticker_internal(
+        self,
+        dataframe: DataFrame,
+        metadata: dict,
+        emit_df: bool = False
```

Comment:
Maybe this needs to be used to protect `self.dp.emit_df` below? As of now it is unused.

Reply:
i don't think this is used anymore - as the leader no longer decides if a dataframe should be sent.
if any, then that check should be in the `dp.emit_df()` method - but i don't think it's necessary - as we moved serialization to the websocket part - only serializing if there's actually a listener.

Reply:
Good catch